components/eamxx/src/share/io/scream_scorpio_interface.F90

```diff
@@ -400,6 +400,10 @@ subroutine register_variable(filename,shortname,longname,units, &
     do while (associated(curr))
       if (associated(curr%var)) then
         if (trim(curr%var%name)==trim(shortname) .and. curr%var%is_set) then
+          ! TODO: For Luca, not sure why this is getting triggered for the time_interpolation test.  We are registering two
```

Comment:
We should remove this check. Our current impl allows to call register_variable multiple times, provided that the metadata is _all_ the same. In fact, in the cpp file, we do perform all these checks.

Notice that we cannot remove the actual call to `register_variable_c2f` in the cpp function if the var is already in the file, since we don't check our F90 metadata, but rather ping the file directly via scorpio calls (so we _must_ call register even if the var is in the file, since we may still need to setup the F90 hist var metadata).

Reply:
The reason why this only appears here is that time interp is the only place that opens the same file twice at the same time (other places close the file before reopening again, like for IC and vert coord).

Reply:
Yes, that's what I figured.  Thanks for confirming.  I'll remove the check.